hw-05/Host/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Models;
+
+string nameAnimal, namePlant;
+int ageAnimal;
+double  sizeAnimal, sizePlant;
+
+Input();
+Animal animal = new Animal(nameAnimal, ageAnimal, sizeAnimal);
+Plant plant = new Plant(namePlant, sizePlant);
+object[] park = new object[5] { Animal._name, Animal._age, Animal._size, Plant._name, Plant._size };
```

Comment:
предполагалось не так, а 
```suggestion
object[] park = new object[2] { animal, plant };
```